extensions/wikia/AssetsManager/config.php

```diff
@@ -3067,3 +3067,11 @@
 		'//extensions/wikia/CrosslinkModule/styles/CrosslinkModule.scss',
 	],
 ];
+
+/* Create new page flow tracking */
```

Comment:
Is it going to be used only for create new page flow or any start & end flow tracking?